fe/fe-core/src/main/java/org/apache/doris/planner/OdbcScanNode.java

```diff
@@ -114,16 +114,22 @@ protected String getNodeExplainString(String prefix, TExplainLevel detailLevel)
         return output.toString();
     }
 
+    // only all conjuncts be pushed down as filter, we can
+    // push down limit operation to ODBC table
+    private boolean needPushDownLimit() {
```

Comment:
```suggestion
    private boolean canLimitPushDown() {
```

Reply:
Here we just represent the `ability` for operator can consumed by our external Table